test/Garnet.test.cluster/Garnet.test.cluster.csproj

```diff
@@ -24,6 +24,7 @@
 
   <ItemGroup>
     <PackageReference Include="CommandLineParser" />
+    <PackageReference Include="Microsoft.CodeAnalysis" />
```

Comment:
why is this package added?

Reply:
I moved the CreateTestLibrary method (that uses CodeAnalysis to build the dlls with the custom classes) to TestUtils since I needed it in a separate test class. Because test.cluster has a link to TestUtils, it also needs that package...